fanficfare/adapters/adapter_silmarillionwritersguildorg.py

```diff
@@ -0,0 +1,236 @@
+        warnText = [warn.string for warn in warnList]
+        self.warnings = ', '.join(charText)
+        for warn in charText:
+            self.story.addToList('warnings',char.string)
+            
+        #get genres
+        genresList = workDetails.findAll('a', href=re.compile(r'browse.php?type=class&type_id=1'+"&classid=\d+$"))
+        genrestext = [genre.string for genre in genresList]
+        self.genre = ', '.join(genrestext)
+        for genre in genrestext:
+            self.story.addToList('genre',genre.string)            
+        
+        #get rating
+        rating = workDetails.find('strong',text='Rated:').next_sibling.string
+        
+        #get completion status and correct for consistency with other adapters
+        if (workDetails.find('strong',text='Completed:').next_sibling.string).lower() == "yes":
+            status="Completed"
+        else:
+            status="In-Progress"
+        
+        #get wordcount
+        wordCount = workDetails.find('strong',text='Word count:').next_sibling.string
+        
+        #get published date, this works for some reason yet doesn't without the spaces in it
+        datePublished = workDetails.find('strong',text=' Published: ').next_sibling.string
+        
+        #get updated date
+        dateUpdated = workDetails.find('strong',text='Updated:').next_sibling.string
+        
+        # try setting metadata values on story, warn if it fails
+        try:
+            self.story.setMetadata('rating', rating)
+            self.story.setMetadata('status', status)
+            self.story.setMetadata('numWords', wordCount)
+            self.story.setMetadata('datePublished', makeDate(datePublished, self.dateformat))
+            self.story.setMetadata('dateUpdated', makeDate(dateUpdated, self.dateformat))
+        except Exception as e:
+            logger.warn("rating, status and/or datePublished parsing failed(%s) -- This can be caused by bad HTML in story description."%e)
+
+    # grab the text for an individual chapter.
+    def getChapterText(self, url):
+
+        logger.debug('Getting chapter text from: %s' % url)
+
+        if self.getConfig('is_adult'):
+            params = {'confirmAge':'1'}
+            data = self._postUrl(url,params)
+        else:
+            data = self._fetchUrl(url)
+
+        soup = self.make_soup(data)
+
+        if "Please indicate that you are an adult by selecting the appropriate choice below" in data:
+            raise exceptions.FailedToDownload("Chapter requires you be an adult.  Set is_adult in personal.ini (chapter url:%s)" % url)
+
+        # No convenient way to get story without the rest of the page, so get whole page and strip unneeded sections
+        
+        contentParent = soup.find('div', {'id' : 'maincontent'}).find('div', {'id' : 'general'})
+        
+        #remove redundant formating tags
+        for tag in contentParent("hr"):
```

Comment:
Removing all hr tags?  Are you sure about that?  Many sites allow (or even require) authors to use hr tags as section separators.

Reply:
My apologies, I must have forgotten to remove this after debugging! 
Thank you for taking the time to give such thorough guidance!